package.json

```diff
@@ -28,25 +28,25 @@
   "license": "MIT",
   "dependencies": {
     "bcrypt-nodejs": "0.0.3",
-    "cheerio": "^0.17.0",
+    "cheerio": "~0.19.0",
     "commander": "^2.3.0",
     "event-stream": "^3.1.7",
     "express": "^4.9.5",
-    "lodash": "~2.4.1",
+    "lodash": "~4.0.0",
     "mkdirp": "^0.5.0",
-    "moment": "~2.7.0",
+    "moment": "~2.11.1",
```

Comment:
For the dependencies you updated, let's use the non tilde-range (`~`) version: `cheerio` in `0.19.0` instead of `~0.19.0`, ...

Having tilde-ranges gives you the false impression you stay latest why this cannot be ensured. I've noticed that projects reaching a decent size switch to the fixed versions as well. Let's do that as we upgrade dependencies :-)